designer/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/BaseDeploymentManager.scala

```diff
@@ -17,6 +17,9 @@ trait BaseDeploymentManager extends DeploymentManager with AlwaysFreshProcessSta
     Future.failed(new UnsupportedOperationException(s"Cannot stop scenario with ${getClass.getSimpleName}"))
   }
 
+  override def stop(name: ProcessName, deploymentId: DeploymentId, savepointDir: Option[String], user: User): Future[SavepointResult] =
+    Future.failed(new UnsupportedOperationException(s"Cannot stop scenario with ${getClass.getSimpleName}"))
```

Comment:
`Cannot stop scenario with base manager`

Reply:
So let's return some human-readable message and leave a log with the class name?